airflow/www/templates/airflow/dags.html

```diff
@@ -78,9 +78,48 @@ <h2>DAGs</h2>
             <th width="12">
               <span id="pause_header" class="material-icons text-muted" title="Use this toggle to pause a DAG. The scheduler won't schedule new tasks instances for a paused DAG. Tasks already running at pause time won't be affected.">info</span>
             </th>
-            <th>DAG</th>
-            <th>Schedule</th>
-            <th>Owner</th>
+            <th>
+              <a href="{{ url_for('Airflow.index', status=request.args.get('status', 'all'), search=request.args.get('search', None), tags=request.args.get('tags', None), sortBy='dag_id', orderBy='asc' if request.args.get('sortBy') != 'dag_id' or request.args.get('orderBy', 'desc') != 'asc' else 'desc' ) }}">
+                DAG
+                <span class="material-icons">
+                  {% if request.args.get('orderBy', 'none') == 'desc' and request.args.get('sortBy') == 'dag_id' %}
+                    expand_less
+                  {% elif request.args.get('orderBy', 'none') == 'asc' and request.args.get('sortBy') == 'dag_id' %}
+                    expand_more
+                  {% else %}
+                    unfold_more
+                  {% endif %}
+                </span>
+              </a>
```

Comment:
This is quite the mouthfull, and repeated three times. Luckily there is `macro` to the rescue.

```jinja
{% macro sorted_th_link(name, attr_name) -%}
               <a href="{{ url_for('Airflow.index',
                                   status=request.args.get('status', 'all'),
                                   search=request.args.get('search', None),
                                   tags=request.args.get('tags', None),
                                   sortBy=attr_name, orderBy='asc' if request.args.get('sortBy') != attr_name or request.args.get('orderBy', 'desc') != 'asc' else 'desc' ) }}"> 
                {{ name }}
                <span class="material-icons">
                  {% if request.args.get('orderBy', 'none') == 'desc' and request.args.get('sortBy') == attr_name %}
                    expand_less
                  {% elif request.args.get('orderBy', 'none') == 'asc' and request.args.get('sortBy') == attr_name %}
                    expand_more
                  {% else %}
                    unfold_more
                  {% endif %}
                </span>
              </a> 
{%- endmacro %}
```

And then we use it like this:

```suggestion
              {{ sorted_th_link("DAG", "dag_id") }}
```

Similarly for the other two headers.

@ryanahamilton Should we set aria tags on the span/link?

Reply:
Ah, yes. Given the text-to-icon strategy employed, we should add `aria-hidden="true"` to the `<span />`.

Reply:
Perhaps also a tooltip to say "Sort by $col desc" too?

Reply:
I've been toying with the idea of slapping a macro onnit, but I wanted to KISS for now. More than happy to refactor it to use the macro. Will add the arias too.

Reply:
Added.